lib/bearerstrategy.js

```diff
@@ -287,7 +287,13 @@ util.inherits(Strategy, passport.Strategy);
 Strategy.prototype.jwtVerify = function jwtVerifyFunc(req, token, metadata, optionsToValidate, done) {
   const self = this;
 
-  const decoded = jws.decode(token);
+  let decoded;
```

Comment:
To be caught in the `if` below, should this be instantiated to `null`? Or have the line 299 be updated to `if (!decoded)`

```suggestion
  let decoded = null;
```